### ISSUE TITLE
Transfer ownership of package to original repo from fork

### ISSUE THREAD

**Issue by martmists** [pypa/warehouse#5648]
Recently I started working on a project with a small team, the repo of which is here, at [martmists/MCServer](https://github.com/martmists/MCServer)

We were still discussing most of the internals when one of the team members left, created their [own fork](https://github.com/BTaskaya/MCServer), broke up all contact and used their fork to claim the [pypi package](https://pypi.org/project/mcserver/) (mcserver) we were aiming to use. We have no way to re-claim this, and we believe claiming our code licensed under all rights reserved without permission violates the license and thus should be removed from pypi for copyright infringement as we are the original license holders.

TL;DR copyright infringement led to an unofficial fork to be published on pypi under an unauthorized person.

**Comment by BTaskaya** [comment]
Any code, content and image that are directly or indirectly linked to your project have been removed from myside. I removed old relases that contains your code from PyPI and github also with that action i started a new project called `MineCraftSERVER` and the abbrevtion of it is `MCServer`. I'm letting PyPI authorities to decide.

**Comment by martmists** [comment]
Despite it being deleted, I still seem to be unable to upload to `mcserver` to claim it.

**Comment by BTaskaya** [comment]
Because i'm the sole owner of name and i  deleted content that directly / indirectly belongs you not my [content](https://github.com/BTaskaya/MineCraftSERVER)

**Comment by martmists** [comment]
You claimed you were fine with it being transferred on discord:
![image](https://user-images.githubusercontent.com/16361449/55278707-0b8ec980-5310-11e9-8641-49361c7ac101.png)

And later said you deleted it per request:
![image](https://user-images.githubusercontent.com/16361449/55278715-1a757c00-5310-11e9-9925-f3e549526acc.png)

I feel like you're being very immature about this, and I hope some Pypa devs can make a final remark on this issue, as you were aware of our intent to publish it to pypi:
![image](https://user-images.githubusercontent.com/16361449/55278733-501a6500-5310-11e9-900d-6626c8a860cd.png)

And now you decide to fully claim the project name for yourself.

**Comment by martmists** [comment]
If anyone believes these images are doctored, here is proof of the messages as well as this being his account:
![info_about_user](https://user-images.githubusercontent.com/16361449/55278780-ea7aa880-5310-11e9-866e-3687e618afbe.gif)

**Comment by BTaskaya** [comment]
You are twisting about my statements. In the first image you submitted i said it is cool to pypi authorities decide any of this expressions says giving the name to you. In the second image i said, i deleted all of your work from PYPI, not deleting the NAME! Deleting relases that contains YOUR CODE!

**Comment by BTaskaya** [comment]
By the way it is a violation of privacy. You are sending all of conversation between you and me without asking my permission. I'm cool with that i stand behind what is said.

**Comment by martmists** [comment]
I am aware that PEP 541 says that disputes like this should be resolved between the users, but in this case:

- You violated copyright, marking your project as invalid
- We then tried to claim it, unable to due to your invalid ownership
- You removed the copyright infringement, still owning the project without ANY codebase

The logical conclusion here is that as we were the first ones to claim ownership without issues, we should gain access to the name, as we were the first to lay valid claim on the name.

It is not a breach of privacy as there was no standard of confidentiality between us, meaning that once sent, it was free to publicize.

**Comment by BTaskaya** [comment]
I didnt violated copyright. It is normal to you can't claim because i'm a hosting my project named "MineCraftSERVER". The project is a name squatting, i'm using the pypi package for my "MineCraftSERVER" project and it has valid code.

**Comment by martmists** [comment]
a class that only inherits asyncio.Protocol and a dataclass with two properties counts as name squatting...

**Comment by BTaskaya** [comment]
(deleted) This repo does not have any relation with `mcserver` package name. The `mcserver` package contains more & more things with time dont expect me to write everything in a hour.

**Comment by martmists** [comment]
```py
from asyncio import Protocol

class MCP(Protocol):
    pass
    
class Server:
    def __init__(self, host, port):
        self.host = host
        self.port = port
```

If you say this as only content does not count as name squatting, go ahead, but I doubt the Pypa members will agree with you. 

That being said, I feel like the Pypa organization members should make a decisive response to this issue as we are clearly unable to figure this out on our own.

**Comment by BTaskaya** [comment]
You are right, as a final comment from my side i'm putting the git logs of old fork. It is a proof of your given LoC changes are lie.
![image](https://user-images.githubusercontent.com/47358913/55279619-874a3f80-532b-11e9-9bc8-ce0d2cf7464c.png)
![image](https://user-images.githubusercontent.com/47358913/55279624-9204d480-532b-11e9-8f1c-746a806edb75.png)

**Comment by Luke-6723** [comment]
Realistically if you do not require the name "MCServer" anymore:

@BTaskaya Quoted as saying: 
> This repo does not have any relation with `mcserver` package name. The `mcserver` package contains more & more things with time dont expect me to write everything in a hour.

why would you have a problem with releasing this name to him. As you stated in a previous comment, You have changed the name to "MineCraftServer":
> Any code, content and image that are directly or indirectly linked to your project have been removed from myside. I removed old relases that contains your code from `PyPI` and github also with that action i started a new project called `MineCraftSERVER` and the abbrevtion of it is `MCServer`. I'm letting PyPI authorities to decide.

 so therefore you should release the name to avoid drama in all cases. If your projects are separate and have nothing to do with each-other why not go your separate ways and just go on instead of making an issue with your projects, since I tend to see programming as a profession and a hobby on my side I wouldn't see why this would be an issue.

**Comment by martmists** [comment]
Marking as resolved, the name has been transferred.

**Comment by BTaskaya** [comment]
I transfered name to @martmists . the only benefit to me was that it gave a good lesson. I teached me never contribute a project without asking the license.

**Comment by martmists** [comment]
More like it taught you:
- not to steal someone elses code and release it under your own name
- not to infringe copyright in the process of doing so
- what the code of conduct for pypi is in regard to name-squatting
